bugsnag-plugin-android-ndk/src/main/assets/include/event.h

```diff
@@ -113,6 +113,8 @@ void bugsnag_app_set_in_foreground(void *event_ptr, bool value);
 
 /* Accessors for event.device */
 
+void bugsnag_device_get_cpu_abi(void *event_ptr, char value[8][32]);
+void bugsnag_device_set_cpu_abi(void *event_ptr, char value[8][32]);
```

Comment:
This feels a little clunky to use with a multi-dimensional array parameter. How about managing the top-level array internally and having get-by-index, size, add and clear methods?